examples/va/hci/edpm-pre-ceph/values.yaml

```diff
@@ -1,30 +1,29 @@
-#
-# CHANGEME:
-#
-# Make the same changes here that you made to openstackdataplanenodeset.yaml from stage 5
-#
-
----
-apiVersion: dataplane.openstack.org/v1beta1
-kind: OpenStackDataPlaneNodeSet
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: openstack-edpm-ipam
-  namespace: openstack
-spec:
-  env:
-    - name: ANSIBLE_CALLBACKS_ENABLED
-      value: profile_tasks
-    - name: ANSIBLE_FORCE_COLOR
-      value: "True"
-  networkAttachments:
-    - ctlplane
-  nodeTemplate:
+  name: edpm-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  ssh_keys:
+    # Authorized keys that will have access to the dataplane computes via SSH
+    authorized: CHANGEME
+    # The private key that will have access to the dataplane computes via SSH
+    private: CHANGEME2
+    # The public key that will have access to the dataplane computes via SSH
+    public: CHANGEME3
+  nodeset:
     ansible:
-      ansiblePort: 22
       ansibleUser: cloud-admin
+      ansiblePort: 22
       ansibleVars:
         edpm_chrony_ntp_servers:
```

Comment:
I think this one doesn't exist anymore. `timesync_ntp_servers` has replaced it and the structure is slightly different:
```yaml
timesync_ntp_servers:
    - hostname: pool.ntp.org
```